change/@fluentui-react-timepicker-compat-preview-1f77ded1-cfbb-4f02-b295-0e08e58e63ff.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30108&commitId=b8618adf83661e24f755553857c0544a1c5fc697&env=prod&clientType=FLUENTUI)


  <details><summary><b>TimePicker compat</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TimePicker compat.Default.default.chromium.png|247| Changed|
|TimePicker compat.Show Seconds.default.chromium.png|247| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>